### PR TITLE
add navbar and router

### PR DESCRIPTION
**Pull request by zantetsuken88**

### CODE REVIEW

File: src/pages/Blog.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+export default class Blog extends Component {
+    render() {
```
Comment (julesnuggy) [inline src/pages/Blog.jsx]: Add styling to this page

File: src/components/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            <>
+                <div className="navbar">
+                <div className="navbar-item">
+                    <Link to="/" className="navbar-link">Home</Link>
+                </div>
+                <div className="navbar-item">
+                    <Link to="/blog" className="navbar-link">Blog</Link>
+                </div>
+                <div className="navbar-item">
+                    <a href="about:blank" className="navbar-link">Services</a>
```
Comment (julesnuggy) [inline src/components/Navbar.jsx]: Create Component Pages for this too

File: src/components/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                </div>
+                <div className="navbar-item">
+                    <a href="about:blank" className="navbar-link">Contact</a>
```
Comment (julesnuggy) [inline src/components/Navbar.jsx]: Create Component Pages for this too

File: src/App.js [hunk 4]
```diff
@@ -1,25 +1,35 @@
...
+                    <div className="header-bar">
+                        <h1>Steven AJ Cox</h1>
+                        <p>Sharing best practice, ideas and opinions from around the world</p>
+                    </div>
+                    <div className="main-body">
+                        <Sidebar/>
+                        <Route exact path="/" component={Home}/>
+                        <Route path="/blog" component={Blog}/>
+                        <Route path="/services" component={Services}/>
+                        <Route path="/contact" component={Contact}/>
```
[the remark this answers is not in the document]
Comment (zantetsuken88) [reply src/App.js]: NOPE!

**Comment by julesnuggy** [comment]
LGTM!

**Pull request merged**